src/EA.Weee.RequestHandlers/Mappings/AatfMap.cs

```diff
@@ -49,7 +49,7 @@ public AatfData Map(Aatf source)
 
             var organisation = organisationMap.Map(source.Organisation);
 
-            return new AatfData(source.Id, source.Name, source.ApprovalNumber, competentAuthority, aatfStatus, address, aatfSize, source.ApprovalDate.GetValueOrDefault())
+            return new AatfData(source.Id, source.Name, source.ApprovalNumber, source.ComplianceYear, competentAuthority, aatfStatus, address, aatfSize, source.ApprovalDate.GetValueOrDefault())
```

Comment:
Check the compliance year property is set, looks like some other properties have been missed in the unit test

Reply:
Added checks for the remaining props